man/clrtrust.1.md

```diff
@@ -0,0 +1,175 @@
+## SYNOPSIS
+
+**clrtrust** is a tool for generating and managing a centralized trusted
+certificate store.
+
+`clrtrust [-v|--verbose] [-h|--help] [-c|--internal-rehash] <command> [options]`
+
+## DESCRIPTION
+
+A trust store contains a set of X.509 certificates which the operating system
+and applications should consider trustworthy. 
+
+The `clrtrust` tool provides a frontend for centralized trust store management.
+It allows for adding (trusting) and removing (distrusting) certificate
+authorities (CAs). It also provides maintenance commands for viewing and
+re-generating the trust store.
+
+Certificates can be provided by the operating system for out-of-box
+functionality. Certificates can also be provided and modified by privileged
+users.
+
+It is up to each application to make use of the trust store generated by
+`clrtrust`.
+
+## OPTIONS
+
+```
+Usage: clrtrust [-v|--verbose] [-h|--help] [-c|--internal-rehash] <command> [options]
+
+    -v | --verbose          Shows more details about execution
+    -c | --internal-rehash  Forces use of internal implementation of c_rehash
+    -h | --help             Prints this message
+
+    Commands
+        generate    generates the trust store
+        list        list CAs
+        add         add trust to a CA
+        remove      remove trust to a CA
+        restore     restore trust to previously removed CA
+        check       sanity/consistency check of the trust store
+
+clrtrust <command> --help to get help on specific command.
+```
+
+**Commands that modify the trust store require root privileges.**
+
+- `clrtrust generate [-f|--force]`
+
+  The `generate` command has no arguments and generates a unified trust store
+  composed of system-provided and user-provided certificates, if any. The
+  optional `--force` parameter will forcibly generate the trust store, even if
+  it results in an empty store. See the FILES section for paths used for trust
+  store generation.
+
+
+- `clrtrust list`
+
+  The `list` command has no arguments and outputs a list of trusted certificates
+  with the following fields:
+
+  `id` uniquely identifies the certificate. It can be used as input to other
+  `clrtrust` commands such as  `remove` or `restore`.
+
+  `File`  contains the file path of the certificate in the trust store. 
+
+  `Authority` shows the name of the organization that issued the certificate.
+    This field is extracted from the certificate file.
+
+  `Expires` shows the expiration date of the certificate. This field is
+    extracted from the certificate file.
+
+
+
+- `clrtust add [<certificateFile|Id> ...] [-f|--force]` 
```

Comment:
One cannot add certificate by id, since it's not known yet to `clrtrust`, only a file is acceptable at this point.